src/docfx/cli/CommandLineOptions.cs

```diff
@@ -43,7 +45,7 @@ internal class CommandLineOptions
 
         public string? TemplateName { get; init; }
 
-        public string? Directory { private get; set; }
+        public string? Directory { get; set; }
```

Comment:
Directory is a command line argument, not an option
```suggestion
        public string? Directory { get; init; }
```

Reply:
Add a  `Input`  option rather than use `Directory`.